packages/github/src/github.ts

```diff
@@ -1,5 +1,6 @@
 // Originally pulled from https://github.com/JasonEtco/actions-toolkit/blob/master/src/github.ts
-import {GraphQlQueryResponse, Variables, defaults} from '@octokit/graphql'
+import { graphql } from '@octokit/graphql'
+import { graphql as GraphQL } from '@octokit/graphql/dist-types/types'
```

Comment:
@thboop not sure if this is the right way to reference this type, since it's imported

Reply:
Talked to Trevor about this and this seems the only way to get to the type unless they start exporting types in the library.